<?xml version="1.0"?>
<launch>

    <arg name="uavId" default="0"/>
    <arg name="missionRadius" default="10.0"/>
    <arg name="missionStep" default="1.0"/>
    <arg name="missionAltitude" default="2.0"/>
    <arg name="minTargetRadiusMeters" default="2.0"/>
    <arg name="serialPort" default="/dev/ttyUSB0"/>
    <arg name="baud" default="57600"/>

    <node pkg="multi_uav_se_mission" name="searcher_manual_node" type="searcher_manual_node" output="screen">
        <!-- parameters -->
        <param name="uavId" value="$(arg uavId)" />
        <param name="missionRadius" value="$(arg missionRadius)" />
        <param name="missionStep" value="$(arg missionStep)" />
        <param name="missionAltitude" value="$(arg missionAltitude)" />
        <param name="minTargetRadiusMeters" value="$(arg minTargetRadiusMeters)" />
        <param name="serialPort" value="$(arg serialPort)" />
        <param name="baud" value="$(arg baud)" />
        <remap from="statistics" to="/uav$(arg uavId)/statistics/data" />
        <remap from="statisticsLog" to="/uav$(arg uavId)/statistics/log" />
        <remap from="statisticsMessages" to="/uav$(arg uavId)/statistics/messages" />
        <remap from="statisticsTargets" to="/uav$(arg uavId)/statistics/targets" />
    </node>

</launch>
